db/migrate/20220810113531_add_unique_to_rewards_title.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueToRewardsTitle < ActiveRecord::Migration[6.1]
```

Comment:
I think `AddUniqueIndexOnTitleToRewards` would be more informative

Reply:
First of all, thanks a lot for review! I have changed this according to your suggestion :)